src/Adapter/Configure/AdvancedParameters/Administration/GeneralConfiguration.php

```diff
@@ -0,0 +1,153 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\Configure\AdvancedParameters\Administration;
```

Comment:
you removed the former `GeneralConfiguration` file and created another one...  thus changing the namespace

This is a breaking change, not sure how big of a deal it is, I'd say it's a big one, but I'd like the opinion of other core maintainers, I'll talk to them 👍

Reply:
Oh yeah, I hadn't thought of that.
Concerning the draft status, it's still relevant since there are other forms to convert in this BO page. I'll try to do it in the week.

Thanks for the feedback!

Reply:
I restored GeneralConfiguration original place, it's not the purpose of the PR I think so it is unnecessary for now to move it.